modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

```diff
@@ -0,0 +1,82 @@
+= Updating the Runtime Version for CloudHub Apps 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: cloudhub, runtime manager, arm, runtime version, mule, updates,
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
+
+To provide a safe cloud environment for applications and data, MuleSoft regularly scans apps deployed on CloudHub and identifies apps that require updates to the CloudHub runtimes.
```

Comment:
@hannanelson We are revisiting the strategy for these updates. So for now, I would like to change this section (up until the snapshot) to reflect the current status and update it again in a couple of week when we have a better schedule.

Reply:
@evinaku - what is the current status?